kolibri/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 #: This may not be the exact version as it's subject to modification with
 #: get_version() - use ``kolibri.__version__`` for the exact version string.
-VERSION = (0, 9, 4, 'final', 0)
```

Comment:
Oh wait... when did we bump to 0.9.4 final? That was never released...

Reply:
Woah, strange, I had understood that it was ready to be released and this is really weird... I'm pretty sure that I did this tag and release, and the tag is reported to exist, I just can't find it in the Release page or in the Tag list.

![image](https://user-images.githubusercontent.com/374612/44720963-b617e300-aac8-11e8-8db8-fc64c074746a.png)

Reply:
https://github.com/learningequality/kolibri/releases/edit/v0.9.4

Reply:
https://github.com/learningequality/kolibri/releases/v0.9.4

Reply:
![image](https://user-images.githubusercontent.com/374612/44721112-2292e200-aac9-11e8-892e-452e9265f14f.png)

Reply:
Why on earth doesn't it show on the Releases list https://github.com/learningequality/kolibri/releases ??

Reply:
That release also says there are 683 commits to release-v0.9.x since that release too... which does not seem right.

Reply:
Can we just delete the tag? There are also no installers on that release.

Reply:
@rtibbles yes, I'll delete the tag from command line, can't actually find it in the Github interface.

Reply:
It is no more!

Reply:
Alright, a correct v0.9.4 is now tagged.

Reply:
This should mean that we can bump the branch by merging this PR. :crossed_fingers: